docs/book/src/topics/ipv6.md

```diff
@@ -0,0 +1,105 @@
+# IPv6 clusters
+- **Feature gate:** ClusterResourceSet=true
+
+## Overview
```

Comment:
Might want to add a note that you should check if your ISP has ipv6 enabled.  I initially ran into this and though was not working. Once connecting from a VM in azure everything worked.